src/server.ts

```diff
@@ -87,9 +102,27 @@ export class Server extends EventEmitter {
    * End all connections and disconnect the plugin
    */
   async close (): Promise<void> {
+    // Stop handling new requests, and return T99 while the connection is closing.
+    // If an F02 unreachable was returned on new packets: clients would immediately destroy the connection
+    // If an F99 was returned on on new packets: clients would retry with no backoff
+    this.plugin.deregisterDataHandler()
+    this.plugin.registerDataHandler(async () => IlpPacket.serializeIlpReject({
+      code: 'T99', // Temporary application error
```

Comment:
nit: instead of error code + comment, this could be `IlpPacket.Errors.codes.T99_APPLICATION_ERROR`.